src/components/types.ts

```diff
@@ -136,11 +136,18 @@ export interface TableFormat extends DataFormat {
 
 export type TableData = TableFormat & Updates<TableFormat>
 
-export interface AudioFormat extends DataFormat {
+export interface MediaFormat extends DataFormat {
   /** URL of the audio file to be played. */
   src: string
   /** URL of the WebVTT captions file (.vtt). */
   captions?: string
   /** Transcript of the audio content. */
   transcript?: string
 }
+
+export interface AudioFormat extends MediaFormat {}
```

Comment:
Question: Is it necessary to add `AudioFormat` when it's exactly same as `MediaFormat`? Do we want to introduce a new type just for the semantic reason?

Reply:
Semantics for clarity but I'm also thinking in the future there could possibly be audio-only fields.